dist/src/main/resources/application.conf

```diff
@@ -23,23 +23,22 @@ sparkta {
     spark.app.name = SPARKTA
     spark.sql.parquet.binaryAsString = true
     spark.streaming.concurrentJobs = 1
```

Comment:
leave these commented lines please. They are useful for the user when he wants to launch the benchmark